build/repo.targets

```diff
@@ -83,4 +113,24 @@
     <Move SourceFiles="%(NPMPackage.OutputTar)" DestinationFiles="%(NPMPackage.ArtifactPath)" />
     <Move SourceFiles="%(NPMPackage.PackageJson).bak" DestinationFiles="%(NPMPackage.PackageJson)" />
   </Target>
+
+  <ItemGroup>
+    <JavaBuildFiles Include="signalr-client-$(JavaClientVersion)-preview1.jar;signalr-client-$(JavaClientVersion)-preview1-javadoc.jar;signalr-client-$(JavaClientVersion)-preview1-sources.jar;signalr-client-$(JavaClientVersion)-preview1.pom"/>
```

Comment:
Can you make the `preview1` part of the file name programmatic too? This is the kind of stuff we typically forget to update when we do preview2.